### PR TITLE
feat: add timing script

### PR DESCRIPTION
**Pull request by cristianmtr**
Add script to run and gather timing information from all examples.

NOTE - only done for adv indexers case, as an example of how it would look like. Pending feedback, should be implemented by all

- [x] advanced-vector-search
- [x] audio-search
- [ ] chinese-text-search
- [ ] cross-modal-search
- [ ] example-guidelines.md
- [ ] fashion-example-query
- [ ] multimodal-search-pdf
- [ ] multimodal-search-tirg
- [ ] multires-lyrics-search
- [ ] ner-search
- [ ] object-search
- [ ] pokedex-with-bit
- [ ] southpark-search
- [ ] templates
- [ ] tumblr-gif-search
- [ ] wikipedia-sentences
- [ ] wikipedia-sentences-incremental

### CODE REVIEW

File: advanced-vector-search/tests/test_advanced_vectors_search.py [hunk 1]
```diff
@@ -8,52 +8,20 @@
...
-    general_config()
-    yield
-    del os.environ['JINA_DATASET_NAME']
-    del os.environ['JINA_USES_FAISS']
-    del os.environ['JINA_USES_ANNOY']
-
-
 @pytest.fixture(scope='session')
-def sift_data(configuration):
+def sift_data():
```
Comment (JoanFM) [inline advanced-vector-search/tests/test_advanced_vectors_search.py:12]: I think `configuration` is needed. Is not about dependency as parameter, but it makes sure that is processed before

**Comment by Yongxuanzhang** [comment]
Shall we merge this one and distribute this task to other people? @cristianmtr @rutujasurve94

File: advanced-vector-search/tests/test_advanced_vectors_search.py (hunk 1 above)
Comment (cristianmtr) [reply advanced-vector-search/tests/test_advanced_vectors_search.py:12]: Seems like it's passing without. Are you sure it's needed?

**Comment by cristianmtr** [comment]
> Shall we merge this one and distribute this task to other people? @cristianmtr @rutujasurve94

Yeah, could be an idea. I started a checklist in this PR. But if we agree to merge this and split it to multiple devs, we could move it to a ticket

**Review by Yongxuanzhang — APPROVED** [review]

**Review by FionnD — APPROVED** [review]

**Pull request merged**